docs/README.md

```diff
@@ -17,6 +17,9 @@ For more information about the Apollo and GraphQL please go to https://www.apoll
 Mutations are persisted when Offline
 and replicated back to server when online.
 
+#### Conflict Resolution
+Provide custom conflict resolution strategies to solve conflicts among the mutations and replicate them back to the server after resolving conflicts.
```

Comment:
```suggestion
Conflict resolution allows providing a mechanism for resolving conflicts between data
sent to server. It will allow developers to specify custom conflict resolution strategies to solve conflicts for mutations and replicate them back to the server after resolution.
```